### PR TITLE
feat: Add font-display swap support to font loading

### PR DESCRIPTION
**Pull request by callinmullaney**
# READY FOR REVIEW

# Summary
- Adds `font-display="swap"` css property to font loading in order to increase site performance and lighthouse scores

# Needed By (Date)
- ASAP

# Urgency
- low

# Steps to Test

1. Review each google font link imported in CSS. Verify the corresponding stylesheets load with the `@font-face` selector including the `font-display: swap` property.

# Affected Projects or Products
- Stanford HumSci platform. Various performance audits recommend this `font-display: swap` value added to `@font-face` in order to increase font loading - see https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display and https://www.smashingmagazine.com/2021/05/reduce-font-loading-impact-css-descriptors/

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by sherakama — APPROVED** [review]
I'm good with this. 

@yvonnetangsu?

**Review by iamrentman — APPROVED** [review]
Looks good to me! Thanks @callinmullaney.

**Comment by yvonnetangsu** [comment]
> Add

Hmm I'm generally ok with this, except for perhaps the Stanford logo font 🤔 I actually have tested the display=swap before, but it creates a flash of different fonts which I think is fine for text, but not sure if we want that for the Stanford logo? It will show the logo as a serif fallback font before the Stanford font finish loading. I don't feel too strongly about it but perhaps UComm branding might have issues with that 🤷🏼‍♀️ 

cc @iamrentman  @callinmullaney

**Comment by iamrentman** [comment]
Thanks @yvonnetangsu. I agree, not with the Stanford ligature font.

File: core/src/scss/core/fonts/_fonts.scss [hunk 1]
```diff
@@ -33,4 +33,5 @@
   ("woff", "ttf")
 ) {
   font-weight: 300;
+  font-display: swap;
```
Comment (iamrentman) [inline core/src/scss/core/fonts/_fonts.scss:36]: Remove from here as this is used for the Stanford logo. As @yvonnetangsu points out, swap can produce a "flash" as the fonts load. Let's prevent that from happening to the logo.

**Comment by callinmullaney** [comment]
Thank you @iamrentman and @yvonnetangsu for taking a look at this pull request. I've removed the `font-display: swap;` from the stanford @font-face. Back to you!

**Comment by yvonnetangsu** [comment]
@sherakama I think this PR is ready to merge but do we have a problem with the PR labeler? I seem to remember you removed it in another repo.

**Comment by sherakama** [comment]
> I think this PR is ready to merge but do we have a problem with the PR labeler? I seem to remember you removed it in another repo.

Yeah, it doesn't have a lot of value for us so I have been removing it.

**Comment by sherakama** [comment]
GTG

**Pull request merged**